index.js

```diff
@@ -7,44 +7,12 @@ function ask(questionText) {
   });
 }
 
+start();
 
 async function start() {
   console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
   let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
   console.log('You entered: ' + secretNumber);
   // Now try and complete the program.
   process.exit();
-}
-
-
-const lowerLimit = 1;
-const upperLimit = 100;
-const tries = 20;
-const answer = math.floor(math.random () * (upperLimit - lowerLimit + 1)) + lowerLimit;
-const guess = '';
-const message = 'Guess a number between ' + lowerLimit + 'and' + upperLimit + ':';
-
-    while (isFinite(guess) && guess != '') {
-        guess = +guess;
-        if (guess < lowerLimit) {
-            alert('Your guess should not be lower than' + lowerLimit + '.');
```

Comment:
instead of using `alert` which only works in the browser use `console.log()` or the `await ask` function we set up for you (if you want to get user input back)